docs/guides/account-resources/user-account-resources.primary.in.rst

```diff
@@ -0,0 +1,168 @@
+It is a requirement defined by the `Science Requirements Document <https://docushare.lsst.org/docushare/dsweb/Get/LPM-17>`_ that the "Data Management System will also provide at least 10% of its total capability for user-dedicated processing and user-dedicated storage" (Section 3.5).
+
+This page quantifies the individual and shared resources that every account at the US Data Access Center (US DAC) has access to, by default, via the Rubin Science Platform at |rsp-url|.
+
+.. important::
+   The US DAC, the Rubin Science Platform, and this documentation are currently under construction.
+   The estimates quoted below include current, planned, and to-be-determined (TBD) values, all of which are subject to change as hardware, software, and user habits evolve.
+
+**Page last updated:** Tue Feb 18 2025
+
+
+Number of users
+===============
+
+Design specification: 10000 individual user accounts.
+
+Number of simultaneous users above which service may degrade:
+
+- Notebook Aspect (JupyterLab servers): 517 :sup:`a`
+- Portal Aspect sessions: TBD
+- API connections: TBD
+
+Maximum number of simultaneous users (hard limit):
+
+- Notebook Aspect (JupyterLab servers): TBD
+- Portal Aspect sessions: TBD
+- API connections: TBD
+
+Maximum number of services accessed simultaneously per user:
+
+- Notebook Aspect (JupyterLab servers): 1 :sup:`b`
+- Portal Aspect sessions: 1 :sup:`b`
+- API connections: TBD
```

Comment:
FYI, the API connections for a user will be based on rate-limit quotas (e.g. requests per amount of time).

Reply:
Added a new footnote to specify that.